crawl-ref/source/dat/descript/de/ability.txt

```diff
@@ -682,110 +681,108 @@ Ihr Ränge in der Fertigkeit Evozieren habt.
 Sacrifice Love ability
 
 Opfert die Liebe und bewirkt, dass alle Kreaturen feindselig werden, auch die
-von Euch erschaffenen. Nur Monster, die von Magie erzeugt sind und nicht
-eigenständig handeln, wie Kugeln der Zerstörung und Spektralwaffen, können
-freundlich bleiben.
+von Euch erschaffenen. Nur Monster, die durch Magie erzeugt werden und nicht
+eigenständig handeln, wie Kugeln der Zerstörung und Spektralwaffen, bleiben
+freundlich.
 %%%%
 Sacrifice Courage ability
 
-Opfert den Mut, so dass Ihr Euch vor gefährlichen Monstern fürchtet. Solche
+Opfert Euren Mut, so dass Ihr Euch vor gefährlichen Monstern fürchtet. Solche
 Monster versetzen Euch in Schrecken, was die Macht Eurer Zauber und Eure
-Fähigkeit, Monster im Kampf zu treffen und ihnen Schaden zuzufügen,
-verringert.
+Fähigkeit verringert, Monster im Kampf zu treffen und ihnen Schaden zuzufügen.
 %%%%
 Sacrifice Arcana ability
 
 Opfert Arkana, wodurch Ihr die Fähigkeit verliert, bestimmte Magieschulen zu
-benutzen und keine Ränge in diesen Fertigkeiten behalten könnt.
+benutzen und alle Ränge in diesen Fertigkeiten verliert.
```

Comment:
You don't just lose current ranks. You also can't train the skill anymore. Same goes for the next two sacrifices.

Reply:
Looked at the English version again. The most literal translation would be:
+ benutzen und keine Ränge in diesen Fertigkeiten haben könnt.

That covers both aspects.